packages/vitest/package.json

```diff
@@ -34,12 +34,16 @@
   },
   "homepage": "https://github.com/seek-oss/sku#readme",
   "devDependencies": {
-    "typescript-transform-paths": "^3.5.2"
+    "typescript-transform-paths": "^3.5.2",
+    "vitest": "catalog:"
   },
   "dependencies": {
     "@vanilla-extract/vite-plugin": "catalog:",
     "vitest": "catalog:",
```

Comment:
Since we're making this a peer dependency, maybe we don't need it as a regular dep anymore?